app/assets/javascripts/articles.coffee

```diff
@@ -0,0 +1,7 @@
+
+do ->
+  onLoad = ->
```

Comment:
Now that we don't have to connect this handler to two different places, you can just inline it below.